packages/neo-one-monitor/src/BrowserMonitor.js

```diff
@@ -90,4 +278,9 @@ export default class BrowserMonitor extends MonitorBase {
       spanLogLevel,
     });
   }
+
+  collect(): MetricCollection {
```

Comment:
Remove, this should only be on the metrics factory itself which gets passed into `create` by the caller.